pkg/api/hvpa_types.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"github.com/gardener/hvpa-controller/api/v1alpha1"
+)
+
+type HvpaConfig struct {
```

Comment:
doc string for all exported types (I don't repeat this for other files, so please check)

Reply:
Doc strings added for all in the api package.